pandasai/smart_datalake/__init__.py

```diff
@@ -206,6 +207,7 @@ def _get_prompt(
         key: str,
         default_prompt: Type[Prompt],
         default_values: Optional[dict] = None,
+        output_type_hint: Optional[str] = None,
```

Comment:
I would pass it as part of default values in the `generate python code` prompt only, so that this is accessible both on the default prompt and in the custom prompts. This might not be needed in every prompt in the future.

Reply:
Sure, you're right, it makes the additional context to be in one place `default_values`. 
Apologize for spreading the setting logic like this, just we've got rly complicated implementation in `._get_prompt()` as for me. As i've mentioned: some variables're being set in `__init__()`, the rest are being set in `_get_prompt()` itself, when iterating over `default_values`. That was kind a tricky moment when i'd had output types mapping inside prompt class before.
___
Done

Reply:
@nautics889 yeah, totally, it's super involved, we'll definitely need to simplify that a little bit in the future 😄